GraphWebhooks/Helpers/SubscriptionStore.cs

```diff
@@ -0,0 +1,42 @@
+﻿/*
+ *  Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license.
+ *  See LICENSE in the source repository root for complete license information.
+ */
+
+using System;
+using System.Web;
+
+namespace GraphWebhooks.Helpers
+{
+    public class SubscriptionStore
```

Comment:
This is great. It makes it obvious what to replace to get a more persistent cache.